project/Settings.scala

```diff
@@ -23,58 +26,113 @@ import com.typesafe.tools.mima.plugin.MimaKeys._
 import com.typesafe.tools.mima.plugin.MimaPlugin._
 import com.typesafe.sbt.SbtScalariform
 import com.typesafe.sbt.SbtScalariform._
-
-import scala.collection.mutable
-import scala.language.postfixOps
-
 import net.virtualvoid.sbt.graph.Plugin.graphSettings
+import com.scalapenos.sbt.prompt.SbtPrompt.autoImport._
+import com.scalapenos.sbt.prompt.PromptTheme
 
 object Settings extends Build {
 
+  val versionStatus = settingKey[Unit]("The Scala version used in cross-build reapply for '+ package', '+ publish'.")
+
   lazy val buildSettings = Seq(
-    name := "DataStax Apache Cassandra connector for Apache Spark",
+    organization         := "com.datastax.spark",
+    version in ThisBuild := "1.2.0-alpha1-SNAPSHOT",
```

Comment:
why the version here is different than below, in settings? is it intentional or just a typo?